### PR TITLE
Add Pikas Miscellaneous Mods

### PR DESCRIPTION
**Pull request by IllusiveMan196**
Apparently Master Index should load after Teleport Warp.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -808,3 +808,28 @@ plugins:
       - 'NOM3.0.esp'
       - 'NOM3_PT SO.esp'
       - 'NoM_Creatures Loot Standard.esp'
+
+  ### Other Stuff (sort later) ###
+  - name: 'Better Propylon Teleport Warp.ESP'
```
Comment (sibir-ine) [inline masterlist.yaml]: The locations & messages can be refactored with a regex plugin:

```suggestion
  - name: 'Better Propylon Teleport Warp(-Master Index)?\.esp'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -808,3 +808,28 @@ plugins:
...
+      - link: 'https://www.nexusmods.com/morrowind/mods/46364/'
+        name: 'Pikas Miscellaneous Mods'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Better Propylon Teleport Warp ESP' ]
+        condition: 'many("Better Propylon Teleport Warp(-Master Index)?\.esp")'
+  - name: 'Better Propylon Teleport Warp-Master Index.esp'
+    url:
+      - link: 'https://www.nexusmods.com/morrowind/mods/46364/'
+        name: 'Pikas Miscellaneous Mods'
```
Comment (sibir-ine) [inline masterlist.yaml]: This can be removed with the refactoring above:

```suggestion
```

File: masterlist.yaml [hunk 3]
```diff
@@ -808,3 +808,28 @@ plugins:
...
+      - 'master_index.esp'
+      - 'MasterIndexRedux.esp'
+    req:
+      - name: 'master_index.esp'
+        condition: 'not active("MasterIndexRedux.esp")'
+        display: '[Master Index Official Plugin](https://www.nexusmods.com/morrowind/mods/42985) or master-index.esp from [Unofficial Morrowind Official Plugins Patched.ESP](https://www.nexusmods.com/morrowind/mods/43931)'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Better Propylon Teleport Warp ESP' ]
+        condition: 'many("Better Propylon Teleport Warp(-Master Index)?\.esp")'
```
Comment (sibir-ine) [inline masterlist.yaml]: This can also be removed with the refactoring above:

```suggestion
```

File: masterlist.yaml [hunk 4]
```diff
@@ -808,3 +808,28 @@ plugins:
...
+  - name: 'Better Propylon Teleport Warp-Master Index.esp'
+    url:
+      - link: 'https://www.nexusmods.com/morrowind/mods/46364/'
+        name: 'Pikas Miscellaneous Mods'
+    after:
+      - 'master_index.esp'
+      - 'MasterIndexRedux.esp'
```
Comment (sibir-ine) [inline masterlist.yaml]: Plugins will automatically load after their requirements, so a load after for `master_index.esp` isn't necessary:

```suggestion
    after: [ 'MasterIndexRedux.esp' ]
```

File: masterlist.yaml [hunk 5]
```diff
@@ -808,3 +808,28 @@ plugins:
...
+    req:
+      - name: 'master_index.esp'
+        condition: 'not active("MasterIndexRedux.esp")'
+        display: '[Master Index Official Plugin](https://www.nexusmods.com/morrowind/mods/42985) or master-index.esp from [Unofficial Morrowind Official Plugins Patched.ESP](https://www.nexusmods.com/morrowind/mods/43931)'
```
Comment (sibir-ine) [inline masterlist.yaml]: The order of the `condition` & `display` keys should be reversed, & I've adjusted the display a bit to be more in line with similar displays in the masterlists:

```suggestion
        display: '[Master Index Plugin](https://www.nexusmods.com/morrowind/mods/42985/) **OR** [Unofficial Morrowind Official Plugins Patched](https://www.nexusmods.com/morrowind/mods/43931/) (**master-index.esp**)'
        condition: 'not active("MasterIndexRedux.esp")'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

File: masterlist.yaml (hunk 4 above)
Comment (IllusiveMan196) [reply masterlist.yaml]: Not all of the Morrowind mods actually have requirements. Or masterfiles at all.

Comment (IllusiveMan196) [reply masterlist.yaml]: Better Propylon Teleport Warp-Master Index.esp has only Morrowind.esm as masterfile and requirement

Comment (sibir-ine) [reply masterlist.yaml]: You added it as a requirement below, though, so it makes a load after redundant. See the `req` section of [this part](https://loot-api.readthedocs.io/en/latest/metadata/data_structures/plugin.html) of the metadata syntax docs.

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**